pkg/scannerv4/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
```

Comment:
I have a bit mixed feeling about a common package constants. Not saying it is bad, but do we use this in other places in the code? I would slightly prefer the old location for them - close to the place where they are used. 
What was the reason for bundling them in a common package?

Reply:
This was a request from Ross back in the other PR (https://github.com/stackrox/stackrox/pull/12310#pullrequestreview-2231038751).
I just realized that I didn't rebase this branch after merging the other one. This has already been merged.